### DISCUSSION TITLE
Mount Limits Equivalent to EQMod

### DISCUSSION THREAD

**Discussion by TonyArchBrown** [rmorgan001/GSServer#33] [Q&A] [answered]
### Equivalent of Mount Limits
Hi there!  I am just in the process of coming over from EQMod to GSS.  Finding it all pretty straight forward so far, connected up to my HEQ5 Pro mount, parking slewing tracking via Cartes du Ciel and Nina and created a new profile in PHD2.  All seems very good I just need to get under some stars to confirm, it's been rain or wind here in my neck of the UK for that 2 weeks.

One of the configurations I found and set  in EQMod was going to the small effort of setting up some Mount Limits.  My understanding was that as commands to slew or track were passing through EQMod it would compare to and take preventative action if one of these limits were being exceeded.  An example of where this saved me from some embarrassment was when I had a NINA advanced schedule running and inadvertently forgot to end the session with a Park or Stop Tracking command, the outcome of which was that NINA completed but the mount kept tracking.  All was well when I returned to the mount the following morning as the Mount Limits had saved me by stopping the mount before the camera would have run into Tripod.  

I see GSS has a Warning system which, if I understand correctly, can be set to prevent tracking if dec is being moved by more than 30d, maybe I am missing some other feature that GSS is doing that EQMod did not and in some way negating the need for Limits.  

Anyway great software and will be trying this out over the coming weeks.  Appreciate anyones thoughts on this.

**Comment by rmorgan001** [comment, marked as answer]
GGS addresses limits differently and has the ability to park, stop, or do nothing when the limits are hit.  Please review the videos on how the limits work at https://greenswamp.org/?page_id=790

**Reply by wanltz** [reply]
Let's unbury this! 

I just got my first GOTO mount, and it's a major concern for me, and i think many people: the ability to avoid a collision between the scope and the mount/tripod. When EQMod can prevent collision in any situation, you can't with GSS, and that's why many people still use this old and outdated EQMod...

EQMod has RA Axis limits, and horizon limits. Both apply in any situation (slew, goto, tracking, etc...).
If the mount pass a limit, EQMod can stop or park the mount.

--------

> GGS addresses limits differently and has the ability to park, stop, or do nothing when the limits are hit. Please review the videos on how the limits work at https://greenswamp.org/?page_id=790

The "Axis limit" in GSS doesn't allow to put negative value for the meridian limit. With my mount and scope, I still can collide, even when tracking. We're many in this situation.
You still can get your mount pointing right to the ground without literally any warning, even when putting 0° for the horizon limit.

That's why he was telling about the "Flip Angle" (Menu -> Settings): it's the only way to set an angle limit on the RA Axis with the counterweight below the scope, and the closest solution to avoir any collision. But the "Flip Angle" just send a warning. You can't set an action when it happens.

--------

We need something that can limit the moves in any situation: tracking, slews and GOTO.

- Be able to set an action when the flip angle warning is triggered, would be enough to avoid collision.

- OR Another solution is the ability to set a negative meridian limit angle, and allowing to apply on both tracking, GOTO, and slews.

- OR Best scenario is the last one + be able to trigger horizontal limits to both tracking, GOTO, and slews.

I'm ready to give a donation for this, or code it myself.

**Reply by rmorgan001** [reply]
Please visit the support site and read up on what's available now.
